index.js

```diff
@@ -13,13 +13,21 @@ function attacher(options) {
   var ignoreMissing = settings.ignoreMissing
   var plainText = settings.plainText || []
   var aliases = settings.aliases
+  var languages = settings.languages
   var name = 'hljs'
   var pos
 
   if (aliases) {
     lowlight.registerAlias(aliases)
   }
 
+  if (languages) {
+    // eslint-disable-next-line guard-for-in
```

Comment:
Disabled eslint here bc this rule doesn't make sense in this context, and adding it in creates some more-or-less untestable surface area that lowers the overall coverage to 98%.

Reply:
Yeah I severely dislike that rule, I typically turn it off in `package.json.xo.rules` when it pops up!